transactions/hybrid-custody/onboarding/blockchain_native.cdc

```diff
@@ -64,22 +64,22 @@ transaction(
         var acctCap = newAccount.linkAccount(HybridCustody.LinkedAccountPrivatePath)
             ?? panic("problem linking account Capability for new account")
 
-        // Create a ChildAccount & link Capabilities
-        let ChildAccount <- HybridCustody.createChildAccount(acct: acctCap)
-        newAccount.save(<-ChildAccount, to: HybridCustody.ChildStoragePath)
+        // Create a OwnedAccount & link Capabilities
+        let OwnedAccount <- HybridCustody.createChildAccount(acct: acctCap)
+        newAccount.save(<-OwnedAccount, to: HybridCustody.ChildStoragePath)
         newAccount
-            .link<&HybridCustody.ChildAccount{HybridCustody.BorrowableAccount, HybridCustody.ChildAccountPublic, HybridCustody.ChildAccountPrivate}>(
+            .link<&HybridCustody.OwnedAccount{HybridCustody.BorrowableAccount, HybridCustody.OwnedAccountPublic}>(
```

Comment:
Should we also link `OwnedAccountPrivate` and/or `MetadataViews.Resolver` here?

Reply:
To the private path, yes I think so, I  must have accidentally deleted the private interface, nice catch 🙏

Reply:
Added `MetadataViews.Resolver`, didn't include `HybridCustody.OwnedAccountPrivate` because it conflicts with `BorrowableAccount`, but it isn't actually needed in this case I think? We need to use `OwnedAccountPrivate` if we do not have key access to the auth account, which we would have. 

We also don't really want every `ChildAccount` resource (formerly `ProxyAccount`) to have full access to all of those methods in the underlying `OwnedAccount. `BorrowableAccount` helps keep what we expose to a minimum.

Thoughts?

Reply:
Right right, that makes sense